server/src/main/java/fr/uga/l3miage/tp2/exo3/Commande.java

```diff
@@ -0,0 +1,20 @@
+package fr.uga.l3miage.tp2.exo3;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import java.util.Date;
+import java.util.Set;
+
+@Entity
+class Commande {
+    @Id
+    private Long id;
+    private Date date;
+    private Double montantTotal;
+    @ManyToOne
+    private Client client;
+    @OneToMany
+    private Set<Produit>produits;
```

Comment:
ici pour la faire de manière unidirectionnelles, il fallait utiliser `@JoinColumn` : 
```suggestion
    @OneToMany
    @JoinColumn(name = "id_produit",referencedColumnName = "id")
    private Set<Produit> produits;
```